java-frontend/src/test/files/se/customresources/OpenedClosed.java

```diff
@@ -19,7 +19,8 @@ public static void correct(String name) {
   
   public static void wrong(String name) {
     GenericResource resource = new GenericResource();
-    resource.open(name);  // Noncompliant
+    // limitation of the current implementation, we don't know the type of resource, so will show "void"
+    resource.open(name);  // Noncompliant [[flows=wrong]] {{Close this "void".}} flow@wrong {{void is never closed}}
```

Comment:
`void` is so bad... Put at least "Resource" if you have no idea about the type, the rule is targeting resources, so it will be at least logical.